tar/storage/packer.go

```diff
@@ -90,11 +90,7 @@ type jsonPacker struct {
 	seen seenNames
 }
 
-type seenNames map[string]byte
-
-// used in the seenNames map. byte is a uint8, and we'll re-use the same one
-// for minimalism.
-const emptyByte byte = 0
```

Comment:
more idiomatic for sets (thought this was in effective go but can't seem to find it now - but http://dave.cheney.net/2014/03/25/the-empty-struct )